packages/gitbook-docs/src/scripts/generateGuidesMetadata.ts

```diff
@@ -23,6 +26,9 @@ const S3_PATH_TO_GITBOOK_DOCS =
   process.env.S3_PATH_TO_GITBOOK_DOCS || 'devportal-docs/docs';
 const S3_GUIDE_METADATA_JSON_PATH =
   process.env.S3_GUIDE_METADATA_JSON_PATH || 'guides-metadata.json';
+const SYNCED_GUIDES_RESPONSE_JSON_PATH = 'synced-guides-response.json';
+const SYNCED_GUIDE_LIST_PAGES_RESPONSE_JSON_PATH =
```

Comment:
Maybe we should have those 2 variables taken from the the .env and having the current value as a fallback in case they are not defined
```suggestion
const SYNCED_GUIDES_RESPONSE_JSON_PATH = process.env.SYNCED_GUIDES_RESPONSE_JSON_PATH || 'synced-guides-response.json';
const SYNCED_GUIDE_LIST_PAGES_RESPONSE_JSON_PATH = process.env.SYNCED_GUIDE_LIST_PAGES_RESPONSE_JSON_PATH || 'synced-guide-list-pages-response.json'
```